examples/index.html

```diff
@@ -194,8 +194,8 @@ <h2 style="margin-top:20px">Save</h2>
   </div>
 
   <footer>
-    <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
-  </footer>
+    * <hr style="margin:20px;"><center><button class="btn btn-default btn-lg" id="clear-cache">Clear offline cache</button></center>
```

Comment:
What does the * do?

Reply:
sorry that was a typo i will make a change :+1: 
Thanks